fastapi/routing.py

```diff
@@ -563,8 +596,25 @@ def include_router(
                     )
         if responses is None:
             responses = {}
+
         for route in router.routes:
             if isinstance(route, APIRoute):
```

Comment:
Reverse the condition and add the continue statement to the if statement to decrease the indentation level.